.gitlab-ci.yml

```diff
@@ -31,7 +31,7 @@ test:
     - chmod +x /usr/local/bin/mender-artifact
   script:
     - cmake -D COVERAGE=ON .
-    - make --keep-going coverage
+    - make --jobs=$(nproc --all) --keep-going coverage
```

Comment:
I usually do '-j -l$NCPUS`, the load is what matters, `-j$NCPUS` hardly every saturates the available resources.

Reply:
Doing a test-run with load set now :) 

Good idea!